rust_wtx_bench/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM docker.io/library/debian:bookworm-slim AS build
+WORKDIR /app
+COPY rust_wtx_bench /app
+COPY proto /app/protos
+RUN apt-get update && apt-get install curl gcc -y
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --default-toolchain nightly-2024-07-10 --profile minimal -y
```

Comment:
Is nightly a strict requirement? Why? If really needed, I'd rather delegate it to the rust toolchain file.